vip-helpers/vip-stats.php

```diff
@@ -40,8 +40,13 @@ function wpcom_vip_top_posts_array( $num_days = 30, $limit = 10, $end_date = fal
 
 		$posts = stats_get_csv( 'postviews', $args );
 	} else {
-		// If Jetpack is not present or not active, fake the data returned
+		// If Jetpack is present and active, but the Stats module has been disabled, add a notice
+		if ( class_exists( 'Jetpack' ) && Jetpack::is_active() && ! Jetpack::is_module_active( 'stats' ) ) {
+
+			_doing_it_wrong( __FUNCTION__, esc_html__( 'The Jetpack Stats module is disabled but used by VIP Stats.' ), '0.1' );
```

Comment:
I don't think _doing_it_wrong() is the right approach here - the calling code for the VIP function is correct, but some other condition at runtime (the Stats module being enabled) is not correct. 

See the description of https://developer.wordpress.org/reference/functions/_doing_it_wrong/ for the intent. 

You wouldn't call _doing_it_wrong() when a MySQLi function was correctly called, but the DB was offline.